solidity/ecdsa/contracts/WalletRegistry.sol

```diff
@@ -390,6 +398,12 @@ contract WalletRegistry is IRandomBeaconConsumer, Ownable {
         //slither-disable-next-line redundant-statements
         misbehavedMembers;
 
+        try walletOwner.notifyEcdsaWalletCreated(publicKeyHash) {} catch {
```

Comment:
Do we want to introduce a gas ceiling for this call similar to what we did in the random beacon callback? This function will be called on the wallet owner contract which we expect to be safe to be called and managed by governance. Introducing a gas limit would introduce complexity and increase the contract size.

Reply:
I don't think we should have the gas limit for the reasons you described. Additionally, two more comments:

1. I do not think we should wrap the notification with a try-catch. Not completing DKG is not a slasheable event and wallet owner is a trusted contract. If the wallet owner was not notified about the wallet created, the entire DKG was for nothing. `notifyEcdsaWalletCreated` should never fail but even if there was a temporary problem in the contract, we could let the `approveDkgResult` to fail and have it submitted one more time later, once that temporary problem in the wallet owner contract gets resolved.
2. Instead of passing `publicKeyHash`, why don't we pass the wallet public key as a parameter? Right now, calling `notifyEcdsaWalletCreated` requires the `Bridge` to call `getWalletPublicKey` on the registry and contracts are interacting back and forth.